analyzer-comments/java/annalyns-infiltration/avoid_unnecessary_parenthesis.md

```diff
@@ -0,0 +1,11 @@
+# Avoid unnecessary parenthesis
```

Comment:
Since its informative, I would suggest less strong language such as "Some parentheses are redundant".

(Note: the plural of parenthesis is parentheses, and in this case you'll always remove two, so plural)

Reply:
Just for my information: the heading in these files is not rendered on the website, right? So this heading is only used to satisfy the "all markdown files should have a level-1 heading" rule enforced by some of the tooling.

Reply:
Looks like the [documentation](https://exercism.org/docs/building/markdown/markdown) answers my question:

> - All files must start start with a level-1 heading (# Some heading text).
> - Level-1 headings exist purely for consuming on GitHub or equivalent.
> - If the file is rendered by Exercism (e.g. displayed on the website, rendered via the CLI), this heading will be removed, and a contextual heading will be inserted.

Reply:
Yeah.

So I actually meant to rename the file, which would usually result in a renamed header ;)